apply/applydriver/local.go

```diff
@@ -49,6 +49,7 @@ type Applier struct {
 	Client              *k8s.Client
 	ImageStore          store.ImageStore
 	CurrentClusterInfo  *version.Info
+	Action              string
```

Comment:
I am afraid that I do not agree that passing a cmd layer type into an underlying procedure, here is the Applier struct.  It makes me feel a bottom struct relies on the upper struct.